uber/templates/emails/mivs/judging_begun.txt

```diff
@@ -8,6 +8,8 @@ Let us know if you have any problems with your entries that may prevent you from
 
 The final date we can accept judging is {{ c.MIVS_JUDGING_DEADLINE|datetime_local }}.  But, we are encouraging you to finish by {{ c.SOFT_MIVS_JUDGING_DEADLINE|datetime_local }}, which will help us tablulate scores on time.  
 
+NOTE: There will be a section in your judging page talking about Video judging.  You can disregard this entire section. 
```

Comment:
Does this mean you'd like me to remove the part of the judging that lets them select a video status? That should be easy to do.